src/cf-layout/src/molecules/heroes.less

```diff
@@ -120,14 +120,14 @@
             }
 
             .m-hero_image {
-                // Remove the flexible container padding so that the image is a consistent width
-                padding-bottom: 0 !important;
+                
+                padding-bottom: 0;
                 height: 100%;
                 width: 100%;
                 position: absolute;
                 background-size: cover;
             }
-        } );
+        } ) !important;
```

Comment:
Oh wild, it's moved it to the whole rule? I guess that works. Hmm.

Reply:
I'm not so sure that's true. I have a feeling the margin top and bottom in the next media query won't work because the rules will now have important attached via this query.